projects/common/xilinx/adi_fir_filter_bd.tcl

```diff
@@ -19,6 +19,7 @@ proc ad_add_decimation_filter {name filter_rate n_chan parallel_paths \
 
   create_bd_pin -dir I $name/aclk
   create_bd_pin -dir I $name/active
+  create_bd_pin -dir I $name/VCC
```

Comment:
Why it is not required to do the same for GND ? 
Perhaps we should look into the ad_connect command to see why can't connect to the VCC anymore.